### ISSUE TITLE
Uninitialized string offset

### ISSUE THREAD

**Issue by conversioaps** [Log1x/sage-directives#58]
When i pass try to pass a post ID into the @field, 
like 
`@field('text', 1)`

I get this error "Uninitialized string offset". 

It look like it try to render `<?= get_field('text')[1]; ?>`

**Comment by dawidurbanski** [comment]
This is the expected behavior.

The second parameter is the key in the array that is used with fields that returns an array. The API is constant for all field types of fields.

```
@field('field_name', 'field_array_key', 'post_id')
```

so you may try to use

```
@field('text', null, 1)
```

**Comment by conversioaps** [comment]
Hi,


Thanks for your response.

If I do as you describe, I then get the error:

"Trying to access array offset on value of type null"

I am using ACF Builder - does it make some issues?


Thanks in advance

Best regards
Simon Andersen

**Comment by conversioaps** [comment]
Or this error:


Med venlig hilsen / Best regards
Simon Andersen
Web Development & Project Manager
Conversio - Digital marketing
Conversio ApS | Hermodsvej 18C | 8230 Åbyhøj
Tel: +45 70 666 500 | D: +45 28 45 61 83E-mail: ***@***.*** | Web: www.conversio.dk
Denne e-mail kan indeholde fortrolige oplysninger.
Hvis du ikke er den tiltænkte modtager af denne e-mail, bedes du omgående slette mailen.

**Comment by conversioaps** [comment]
I think it's because my code editor does not recognize the format of the parameters correctly. 
How do I fix it? I use PhpStorm.

<img width="449" alt="Skærmbillede 2021-03-20 kl  09 42 33" src="https://user-images.githubusercontent.com/8332607/111864351-a3a07600-8960-11eb-8f78-e93faa1a76d8.png">

**Comment by Log1x** [comment]
Fixed in #82
